pkg/operator/register.go

```diff
@@ -13,14 +16,32 @@ import (
 	"github.com/Layr-Labs/eigenlayer-cli/pkg/utils"
 	elContracts "github.com/Layr-Labs/eigensdk-go/chainio/clients/elcontracts"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
-	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
 	eigensdkLogger "github.com/Layr-Labs/eigensdk-go/logging"
 	"github.com/Layr-Labs/eigensdk-go/metrics"
-	eigensdkUtils "github.com/Layr-Labs/eigensdk-go/utils"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/urfave/cli/v2"
 )
 
+type ChainMetadata struct {
+	BlockExplorerUrl           string
+	ELDelegationManagerAddress string
+}
+
+var chainMetadataMap = map[int64]ChainMetadata{
+	utils.MainnetChainId: {
+		BlockExplorerUrl:           "https://etherscan.io/tx",
+		ELDelegationManagerAddress: "",
+	},
+	utils.GoerliChainId: {
+		BlockExplorerUrl:           "https://goerli.etherscan.io/tx",
+		ELDelegationManagerAddress: "0x1b7b8F6b258f95Cf9596EabB9aa18B62940Eb0a8",
```

Comment:
@ypatil12 can you confirm this is right?

Reply:
Look good